src/evalml/config.py

```diff
@@ -86,6 +90,29 @@ class RunConfig(BaseModel):
 
     config: Dict[str, Any] | str
 
+    @field_validator("steps")
+    def validate_steps(cls, v: str) -> str:
+        if "/" not in v:
+            raise ValueError(
+                f"Steps must follow the format 'start/stop/step', got '{v}'"
+            )
+        parts = v.split("/")
+        if len(parts) != 3:
+            raise ValueError("Steps must be formatted as 'start/end/step'.")
+        try:
+            start, end, step = map(int, parts)
+        except ValueError:
+            raise ValueError("Start, end, and interval must be integers.")
+        if start >= end:
+            raise ValueError(f"Start ({start}) must be less than end ({end}).")
+        if step <= 0:
+            raise ValueError(f"Interval ({step}) must be a positive integer.")
+        if (end - start) % step != 0:
```

Comment:
The help above says that the definition is `[start, end)`, so this should be equivalent `0/126/6` to `0/121/6`, but it is not. We either need to adjust the documentation or change the paradigm. I am (obviously) much in favor for intervals including the end point (if matching with steps) such that we can just specify `0/120/6` for COSMO-(2)E and `0/33/6` for COSMO-1(E) without thinking to hard ;-).

Reply:
agreed!